<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>viewerBoxes3D</class>
 <widget class="QWidget" name="viewerBoxes3D">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>800</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>1000</width>
    <height>800</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1000</width>
    <height>800</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>在线编辑</string>
  </property>
  <layout class="QGridLayout" name="gridLayout" columnstretch="2,1">
   <item row="0" column="0">
    <widget class="QWidget" name="widget_3DViewer" native="true">
     <layout class="QGridLayout" name="gridLayout_4"/>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QWidget" name="widget" native="true">
     <layout class="QGridLayout" name="gridLayout_2" rowstretch="2,10">
      <item row="0" column="0">
       <widget class="QWidget" name="widget_2" native="true">
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QWidget" name="widget_3" native="true">
           <layout class="QGridLayout" name="gridLayout_3">
            <property name="leftMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>0</number>
            </property>
            <item row="0" column="0" colspan="2">
             <widget class="QComboBox" name="comboBox"/>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QWidget" name="widget_6" native="true">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>50</height>
            </size>
           </property>
           <layout class="QGridLayout" name="gridLayout_6">
            <item row="1" column="2">
             <widget class="QPushButton" name="pushButton_returnHome">
              <property name="font">
               <font>
                <pointsize>15</pointsize>
               </font>
              </property>
              <property name="text">
               <string>登出</string>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QPushButton" name="pushButton_returnChooseMode">
              <property name="font">
               <font>
                <pointsize>15</pointsize>
               </font>
              </property>
              <property name="text">
               <string>返回模式选择</string>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QPushButton" name="pushButton_disconnectPLC">
              <property name="font">
               <font>
                <pointsize>15</pointsize>
               </font>
              </property>
              <property name="text">
               <string>断开PLC</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QWidget" name="widget_4" native="true">
           <layout class="QVBoxLayout" name="verticalLayout_2">
            <item>
             <widget class="QCommandLinkButton" name="createCubeButton">
              <property name="text">
               <string>CommandLinkButton</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCommandLinkButton" name="readFileButton">
              <property name="text">
               <string>CommandLinkButton</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCommandLinkButton" name="readSqlButton">
              <property name="text">
               <string>CommandLinkButton</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QWidget" name="widget_5" native="true">
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <widget class="QLabel" name="label">
           <property name="text">
            <string>栈板长度(dm):</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lengthLineEdit"/>
         </item>
         <item>
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>栈板宽度(dm):</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="widthLineEdit"/>
         </item>
         <item>
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>栈板高度限制(dm):</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="heightLineEdit"/>
         </item>
         <item>
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>货物长度(dm):</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="goods_length_LineEdit"/>
         </item>
         <item>
          <widget class="QLabel" name="label_5">
           <property name="text">
            <string>货物宽度(dm):</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="goods_width_LineEdit"/>
         </item>
         <item>
          <widget class="QLabel" name="label_6">
           <property name="text">
            <string>货物高度(dm):</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="goods_height_LineEdit"/>
         </item>
         <item>
          <widget class="QLabel" name="label_7">
           <property name="text">
            <string>超托率(0~1):</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="outRatioLineEdit"/>
         </item>
         <item>
          <widget class="QLabel" name="label_8">
           <property name="text">
            <string>单货物重量(kg):</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="weightLineEdit"/>
         </item>
         <item>
          <widget class="QLabel" name="label_9">
           <property name="text">
            <string>总货物重量(kg):</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="totalweightLineEdit"/>
         </item>
         <item>
          <widget class="QDialogButtonBox" name="buttonBox">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="standardButtons">
            <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
